pyam/core.py

```diff
@@ -1047,30 +1047,49 @@ def require_variable(self, *args, **kwargs):
         # TODO: deprecated, remove for release >= 2.1
         raise DeprecationWarning("Use `df.require_data()` instead.")
 
-    def validate(self, criteria={}, exclude_on_fail=False):
-        """Validate scenarios using criteria on timeseries values
+    def validate(
+        self,
+        criteria: dict = None,
+        *,
+        upper: float = None,
+        lower: float = None,
```

Comment:
Same as above comment 
```suggestion
        lower_bound: float = None,
```